### DISCUSSION TITLE
My code clearly does not call this service, why does the skywalking topology still show me calling it

### DISCUSSION THREAD

**Discussion by wang1000001000** [apache/skywalking#11400] [Q&A]
This situation occurs many times and the corresponding link cannot be found

**Comment by wu-sheng** [comment]
Your question is too abstract. The only guess I have, did they share the same address? If not, it is impossible to get that.

**Reply by wang1000001000** [reply]
![image](https://github.com/apache/skywalking/assets/89133141/ffb88684-3794-4f7b-95f1-41697ae5c3e6)Two separate services There are two separate services, both of which have clear traffic, but it's very rare.  I can't find the specific trace.  What I can confirm is that there is no call to these services in the code, and the services are independent of each other.

**Reply by wu-sheng** [reply]
If you are talking in this way, I don't know what help I can offer.
